ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -39,6 +39,13 @@ public protocol BottomSheetControllerDelegate: AnyObject {
     case transitioning // Sheet is between states, only used during user interaction / animation
 }
 
+/// Defines where the sheet should be postionioned relative to the screen space
+@objc public enum BottomSheetAnchorEdge: Int {
```

Comment:
Sorry I missed this before, but does it need to be @objc(MSFBottomSheetAnchorEdge) to get the proper class prefix for objC usage?